### ISSUE TITLE
Revealjs: Rows entering after the pause are ignored by the plot size and pushed out of the screen

### ISSUE THREAD

**Issue by eitsupi** [quarto-dev/quarto-cli#1471]
### Bug description

If the following slide displays text below the plot, setting a pause in front of the plot will push the text off the screen.

````qmd
---
title: sample
format:
  revealjs: default
engine: knitr
---

## page1

foo

. . .

```{r}
plot(mtcars)
```

bar

## page2

foo

```{r}
plot(mtcars)
```

bar

````

![image](https://user-images.githubusercontent.com/50911393/180027495-f48fefde-9fa6-448f-97c6-8f68c03a295c.png)
![image](https://user-images.githubusercontent.com/50911393/180027553-e4cf0290-de54-4c20-a0d8-0a06d1c0d1fa.png)

### `quarto check` Output

```sh
[✓] Checking Quarto installation......OK
      Version: 1.0.35
      Path: /opt/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.8.10
      Path: /usr/bin/python3
      Jupyter: 4.10.0
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.2.0
      Path: /usr/local/lib/R
      LibPaths:
        - /usr/local/lib/R/site-library
        - /usr/local/lib/R/library
      rmarkdown: 2.14

[✓] Checking Knitr engine render......OK
```

### `quarto tools check` Output

[✓] Inspecting tools

Tool         Status            Installed     Latest  
chromium     Not installed     ---           869685  
tinytex      Not installed     ---           v2022.07

### Checklist

- [X] [formatted your issue](https://yihui.org/issue/#please-format-your-issue-correctly) so it is easier for us to read?
- [X] included a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] documented the quarto version you're running, by pasting the output from running `quarto check` in the "Quarto Check Output" text area?
- [X] documented the version of the quarto tools you're running, by providing the output from running `quarto tools check` in the "Quarto Tools Check Output" text area?
- [ ] documented the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [ ] documented which operating system you're running? If on Linux, please provide the specific distribution as well.
- [X] upgraded to the latest version, including your versions of R, the RStudio IDE, and relevant R packages?

**Comment by jjallaire** [comment]
Normally plots are auto-sized by reveal but if they are initially hidden the can't be (as reveal actually measures the DOM size as part of its auto-sizing). In this case you need to manually size the plot.

**Comment by cderv** [comment]
yes, stretch feature and fragment in revealjs does not work together. For reference, related to https://github.com/quarto-dev/quarto-cli/issues/816 and upstream https://github.com/hakimel/reveal.js/issues/1840

**Comment by eitsupi** [comment]
Thanks for your quick reply!
It may be worth mentioning this behavior in the reference page.
